jet_tracking/devices.py

```diff
@@ -746,6 +746,28 @@ class Parameters(Device, _TableMixin):
                          doc='Nozzle counter')
     nozzle_reprate = Cpt(EpicsSignal, ':NOZZLE_RepRate',
                          doc='Nozzle repetition rate')
+    mean = Cpt(EpicsSignal, ':ROI_mean',
+               doc='mean of calibration ROI image with jet')
+    std = Cpt(EpicsSignal, ':ROI_std',
+              doc='standard devation of calibration ROI image with jet')
+    radius = Cpt(EpicsSignal, ':RADIUS',
+                 doc='radius of calibration diffraction ring')
+    intensity = Cpt(EpicsSignal, ':INTENSITY',
```

Comment:
So here `intensity` is spelled out, and below some thresholds are defined that don't really indicate what they are thresholding.  See other comment about variable names for intensity.

Reply:
I strongly suspect that these are the PVs for the results of the calibration. So (handwaving starts.... we take a calibration run, and run our fancy 'jet_tracking_<SAXS/Ring/...>_calibrate.ipynb' which comes up with inputs to the jet tracking as the mean radius (and/or min&max radii) for the azimuthal average and the intensity & i_zero (or only ratio?) we should expect. This gets written to a file in the offline calibration dir. We start the psana code which reads this calibration file and pushes these values to PVs: this is to that the GUI can show then and that the jet tracking Ophyd layer as the "raw" information available (calibration results as well as current results) to make decisions. ...handwaving end)